packages/web/config/price.ts

```diff
@@ -480,4 +480,24 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     ),
     destCoinId: "stargaze",
   },
+  {
+    alternativeCoinId: "pool:sejuno",
```

Comment:
can we please add these alt coin ids to the chain-infos.tx file? (as: "coinGeckoId": "pool:sejuno", and "coinGeckoId": "pool:sejuno", in their respective token definitions)

Reply:
Fixed, thanks for pointing out.